look/__brick__/look/look_data/specific_look_data/color_look_data.dart

```diff
@@ -1,43 +1,45 @@
 import 'package:flutter/material.dart';
+import 'package:look/ui/common/look/look_data/specific_look_data/colors/neutral_color.dart';
 
-class ColorLookData {
-  const ColorLookData({
-    required this.brightness,
-    required this.primary,
-    required this.secondary,
-    required this.onPrimary,
-    required this.onSecondary,
-    required this.background,
-    required this.surface,
-    required this.onBackground,
-    required this.onSurface,
-    required this.error,
-    required this.onError,
-  });
+class ColorLookData extends ThemeExtension<ColorLookData> {
+  const ColorLookData({required this.attentionColor, required this.neutral});
 
   const ColorLookData.getDefault()
-      : brightness = Brightness.light,
-        primary = const Color(0xff005670),
-        secondary = const Color(0xffAA198D),
-        background = Colors.white,
-        surface = Colors.white,
-        error = const Color(0xffFB3449),
-        onError = Colors.white,
-        onBackground = const Color(0xff666666),
-        onSurface = const Color(0xff666666),
-        onPrimary = Colors.white,
-        onSecondary = Colors.white;
+      : attentionColor = const Color(0xfffdbf2f),
+        neutral = const NeutralColor(0xFF12335B, {
+          100: Color(0xFF12335B),
+          80: Color(0xFF415C7C),
+          60: Color(0xFF71859D),
+          40: Color(0xFFA0ADBD),
+          30: Color(0xFFD0D6DE),
+          20: Color(0xFFE9ECF0),
+          10: Color(0xFFF5F6F7),
+          0: Color(0xFFFFFFFF),
+        });
 
-  // Material color scheme
-  final Brightness brightness;
-  final Color primary;
-  final Color secondary;
-  final Color onPrimary;
-  final Color onSecondary;
-  final Color background;
-  final Color surface;
-  final Color onBackground;
-  final Color onSurface;
-  final Color error;
-  final Color onError;
+  // Color not in material scheme
+  final Color attentionColor;
+
+  // Custom color swatch not in material scheme
+  final NeutralColor neutral;
+
+  @override
+  ColorLookData copyWith({Color? attentionColor, NeutralColor? neutral}) {
+    return ColorLookData(
+      attentionColor: attentionColor ?? this.attentionColor,
+      neutral: neutral ?? this.neutral,
+    );
+  }
+
+  @override
+  ThemeExtension<ColorLookData> lerp(ThemeExtension<ColorLookData>? other, double t) {
+    if (other is! ColorLookData) {
+      return this;
+    }
+
+    return ColorLookData(
+      attentionColor: Color.lerp(attentionColor, other.attentionColor, t) ?? other.attentionColor,
+      neutral: other.neutral,
+    );
+  }
```

Comment:
Do you know any use case when we would  need this?

Reply:
Something like this maybe:

![Screenshot 2022-10-12 at 16 02 41](https://user-images.githubusercontent.com/59824861/195363988-f2c16a50-cfb6-4d72-b8bc-26bb01bb68a4.png)